osaccountmanager.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"net/http"
+	"os"
+)
+
+// Implements the AccountManager interface taking the username from the
+// environment and authorizing all requests
+type OsAccountManager struct{}
```

Comment:
is this account manager meant for development purposes only? if so, let's add it to the documentation.

Reply:
We'll use it for that purpose only, but others are free to use it for something else if it makes sense.